### DISCUSSION TITLE
Shared Experience?

### DISCUSSION THREAD

**Discussion by FishGaming** [diasurgical/DevilutionX#6292] [Q&A]
I was so excited about this QoL improvement mod. Me and three friends started playing and no one is gaining experience at an equal rate. Is experience not shared in this mod?!?! If so that would be a huge oversight for this amazing mod. Please tell me I'm missing that gameplay feature somewhere in settings or in an ini file.

**Comment by DakkJaniels** [comment]
Shared experience is not included.

**Comment by AJenbo** [comment]
> If so that would be a huge oversight for this amazing mod.

Or simply a difference in opinion. We generally do not change how gameplay works

**Comment by FishGaming** [comment]
I meant it's an oversight that the option was not there, not that it wasn't just given all the time. I'm still working on trying to compile the source code though that you guys started me on the forum. Hopefully I can understand this stuff enough to make it happen!

**Reply by AJenbo** [reply]
I realized that. But yeah it's not so different from the friendly fire option, so it's not totally out of scope, which I think @kphoenix137 will appreciate. If you get stuck at all or are confused about how to get things to build just ask on the chat and we can help you with that part.
Did you notice that KP also posted a prebuild test version for you that you can just grab and start playing?

**Reply by FishGaming** [reply]
No I didn't! So with the prebuild test version, would I just need to use visual studio to edit the lines he spoke about? Where is this located?

**Reply by FishGaming** [reply]
Actually I just found his merge pull request on his profile. How do I download this version to play?
